libraries/liveIntentId/idSystem.js

```diff
@@ -92,7 +93,11 @@ function initializeLiveConnect(configParams) {
   const publisherId = configParams.publisherId || 'any';
   const identityResolutionConfig = {
     publisherId: publisherId,
-    requestedAttributes: parseRequestedAttributes(configParams.requestedAttributesOverrides)
+    requestedAttributes: parseRequestedAttributes(configParams.requestedAttributesOverrides),
+    extraAttributes: {
+      ipv4: configParams.ipv4,
```

Comment:
why make these config params instead of just gathering from where the publisher already set them? in ortb2.device.ip and ipv6?

Reply:
I will get back to you

Reply:
Hi @patmmccann, `ortb2` seems currently not to be accessible for the user ID submodules. That was the reason for implementing it that way. If you are fine with it, we can change that and pass `ortb2` to every user ID submodule during initialization.